src/main/java/uk/gov/digital/ho/hocs/workflow/api/dto/CreateCaseRequest.java

```diff
@@ -21,6 +24,7 @@
     private LocalDate dateReceived;
 
     @JsonProperty("data")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Again i think this is a residual left from my branch. It's not necessary as the front end always sends back at least `data = {}`.

Reply:
I have addressed this.